recipe/meta.yaml

```diff
@@ -1,70 +1,77 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.8.3.1" %}
+{% set sha256 = "4b0f5a5b4828915d973d3cbb86372aea9207e7b60fed912045d69242a74407f6" %}
+
 package:
-    name: ulmo
-    version: 0.8.2
+  name: ulmo
+  version: {{ 0.8.3.1 }}
```

Comment:
That is correct. We can run the tests, but usually we trust they are run upstream and that we only need to check if the package can be "used."

The main reason is to avoid an extra burden on the CIs, but if the tests are light weight and run under the CIs time limit feel free to add them here.